test/e2e/specs/plans/plans__signup-starter.ts

```diff
@@ -0,0 +1,186 @@
+/**
+ * @group calypso-release
+ */
+
+import {
+	DataHelper,
+	BrowserManager,
+	SecretsManager,
+	LoginPage,
+	UserSignupPage,
+	SignupPickPlanPage,
+	StartSiteFlow,
+	SidebarComponent,
+	PlansPage,
+	RestAPIClient,
+	CartCheckoutPage,
+	DomainSearchComponent,
+	IndividualPurchasePage,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+import type { NewSiteDetails } from '@automattic/calypso-e2e';
+
+declare const browser: Browser;
+
+describe(
+	DataHelper.createSuiteTitle( 'Plans: Create a WordPress.com Starter site as new user' ),
```

Comment:
I like the pairwise-esque design here with this being a new user, and the Pro being an existing user. 🙂